### ISSUE TITLE
MessageMedia not sending as a voice message

### ISSUE THREAD

**Issue by byeze** [pedroslopez/whatsapp-web.js#207]
Hello! I'm using this code for sending an audio as a voice message:
```
var audio = fs.readFileSync("./p/audio.mp3", {
    encoding: "base64",
});
var media = new MessageMedia("audio/mpeg", audio);
msg.reply(media, null, { sendAudioAsVoice: true });
```
But the message is sending as an audio, not respecting the sendAudioAsVoice option.
![image](https://user-images.githubusercontent.com/7424138/82499639-682ae600-9ac8-11ea-80b4-9995198ed91f.png)

**Comment by pedroslopez** [comment]
Hi byeze, this functionality is not currently released on the npm package and will be included in the next release. If you need this urgently, you can load the library from GitHub or just wait until this weekend when I'll hopefully release the next version.

**Comment by biscoitin** [comment]
I have used downloadMedia like image bellow:

![codeWpp](https://user-images.githubusercontent.com/1566736/82556440-145be380-9b40-11ea-8ac0-1e52430310be.png)

**Comment by biscoitin** [comment]
But it didn`t work as well between 09:00 AM and 12:00 AM yesterday

**Comment by pedroslopez** [comment]
@byeze This has now been released in v1.6.0
